prow/plugins.yaml

```diff
@@ -356,7 +356,11 @@ plugins:
 
   kubernetes/features:
   - approve
+  - blockade
```

Comment:
There are no blockades (see earlier in the file) configured for this repo so this won't do anything. Did you mean to enable `blunderbuss`?

Reply:
There aren't blockades yet, but there will be, to prevent writes to historical dirs.

Reply:
I've added `blunderbuss` as well, as that will be required soon.